src/server/service/graphql/schema/queries/mod.rs

```diff
@@ -6,12 +6,18 @@ use crate::database::schema::{
     ItemLineRow, ItemRow, NameRow, RequisitionRow, StoreRow, TransactLineRow, TransactRow,
 };
 use crate::server::service::graphql::schema::types::{
-    Item, ItemLine, Name, Requisition, Store, Transact, TransactLine,
+    Item, ItemLine, Requisition, Store, Transact, TransactLine,
 };
 use crate::server::service::graphql::ContextExt;
 
 use async_graphql::{Context, Object};
 
+use self::pagination::Pagination;
+
+use super::types::NameList;
+
+pub mod pagination;
+
```

Comment:
I think the pagination import can be made a bit cleaner by moving it after the module declaration?

```suggestion
use super::types::NameList;

pub mod pagination;

use pagination::Pagination;
```

Reply:
760c2a9ca0d2910110daf368df17169b67ab19d7